lang/intlMom.js

```diff
@@ -1,9 +1,28 @@
 export default {
-  de: "Hallo, {{Mama}}!",
   en: "Hi, {{mom}}!",
+  de: "Hallo, {{Mama}}!",
   "en-AU": "Hi, {{mum}}!",
   fr: "Salut, {{maman}}!",
   it: "Ciao {{mamma}}!",
   pt: "Oi, {{mãe}}!",
   ru: "Привет, {{мама}}!",
+  "nl-NL": "Hallo, {{moeder}}!",
+  "nl-BE": "Hallo, {{moeke}}!",
+  cn: "你好，{{妈妈}}!",
+  es: "Hola, {{mamá}}!",
+  ja: "こんにちは、{{ママ}}！",
+  ko: "안녕하세요, {{마모}}!",
+  tr: "Merhaba, {{baba}}!",
+  th: "สวัสดีค่ะ, {{มา}}!",
+  vi: "Xin chào, {{mẹ}}!",
+  id: "Hai, {{ibu}}!",
+  ms: "Hai, {{bapa}}!",
+  ca: "Hola, {{mamà}}!",
+  cs: "Ahoj, {{mamá}}!",
+  he: "שלום, {{מה}}!",
+  hu: "Szia, {{anyá}}!",
+  pl: "Cześć, {{matka}}!",
+  ro: "Bună, {{mamă}}!",
+  uk: "Привіт, {{мама}}!",
```

Comment:
uk is for United Kingdom. for Ukraine is ua

Reply:
Oh my bad, should be fixed now